library/core/src/hint.rs

```diff
@@ -108,6 +106,8 @@ pub fn spin_loop() {
 /// Note however, that `black_box` is only (and can only be) provided on a "best-effort" basis. The
 /// extent to which it can block optimisations may vary depending upon the platform and code-gen
 /// backend used. Programs cannot rely on `black_box` for *correctness* in any way.
+///
+/// [`std::convert::identity`]: ../convert/fn.identity.html
```

Comment:
`convert` is in core, so you can use intra-doc links here:
```suggestion
/// [`std::convert::identity`]: crate::convert::identity
```